buildscripts/generate-release-contributors.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
```

Comment:
This file doesn't seem to be documented, intended to add it?

Reply:
sort of intentional, I wanted to get feedback on whether we should formalize this as part of the processes, but better to remove from this PR and add to SIG agenda